src/locale/languages/de.js

```diff
@@ -1,7 +1,43 @@
-// Hash: 01ae4880b9b4296907d680e9b909713c36c42491
-// Link: https://github.com/gitpoint/git-point/pull/260
-
-export const de = {
+module.exports = {
```

Comment:
In order to be able to read/write the file from the extraction utility, I had to change the export syntax.

This made things simpler in src/locale/index.js